docs/src/main/sphinx/connector/iceberg.md

```diff
@@ -1529,6 +1529,121 @@ the definition and the storage table.
 The connector supports {doc}`/admin/fault-tolerant-execution` of query
 processing. Read and write operations are both supported with any retry policy.
 
+## Table functions
+
+The connector provides the following table functions:
+
+### table_changes
+
+Allows reading row-level changes between two versions of an Iceberg table. 
+This is how these changes can be read:
+
+```sql
+SELECT
+  *
+FROM
+  TABLE(
+    system.table_changes(
+      schema_name => 'default',
+      table_name => 't1',
+      start_snapshot_id => 6541165659943306573,
+      end_snapshot_id => 6745790645714043599
+    )
+  );
+```
+
+`schema_name` - type `VARCHAR`, required, name of the schema for which the function is called
```

Comment:
I think we can leave the type out for all these parameters. Also maybe use the same list layout as in the following section.